### ISSUE TITLE
UI/UX improvements to Summary page

### ISSUE THREAD

**Issue by hathix** [trynmaps/metrics-mvp#319]
On #314, Jesse pointed out:

Thanks for doing this, I think the Summary tab looks much better.

> On the Summary tab, "Typical Trip" seems like the most important stat, so it seems like it should be first. Maybe this stat shouldn't include the wait time, though. A lot of people would plan their trips based on the predicted arrival time so they wouldn't actually be waiting the median wait time. This would also avoid needing to show how the typical trip stat is calculated as the sum of wait time and trip time (e.g. "14 min wait + 8 min"), which could be confusing to users
    I think people would be confused by the stat labeled "90% of trips". Maybe instead of having a different card for this, the Typical Trip card could show something like "90% of trips less than XX minutes" underneath the median trip time.
    The median wait stat info bubble could explain that this is if you arrive at the origin stop at a random time (ignoring schedule/predictions)
    It's probably unclear to users what "Extra travel" card means .... took me a while to figure out that this is the 90th percentile trip time minus the median trip time. It seems like it would be hard to explain this in a way that does not confuse or overwhelm people. I'd prefer to remove Extra Travel from the summary and the route table to keep it simple and reduce the information overload. Maybe there could be an advanced stats tab for things like this, or it could just be in the Trip Score info bubble.
    "20 Min Wait" also seems like it could be confusing to people (probability of having to wait 20 minutes if you show up at a random time ignoring schedule/predictions). Maybe this could go in an advanced tab rather than being in the summary tab and route table.
> Maybe the cards on the Summary tab could be Trip Time, Wait Time, Speed, Trip Score? I think reducing the number of top line stats like this would make it less overwhelming for users.

**Comment by exxonvaldez** [comment]
I've passed this issue along to Ricky to see if it can be worked into the usability test.

NYC bus stats shows the same 4 numbers (planning wait/travel time, average wait/travel time), but as a bar chart.  That chart is what we modeled our intervals of the day chart on.  So we could change those two cards to the bar chart format.

For scenarios where I want to get to a place before a specific time, the 90% stats are more important than median.  The 90% wait time gives me an idea of when to start checking predictions, and it can be a much higher number than I expect.  Too often, I've started to check predictions at about (walk time to stop + scheduled headway) before when I would like to board, only to realize due to gaps in service that I'm going to be late.

I think long waits (still a placeholder for how many gaps there are) and travel variance are useful for characterizing a line in a way that wait and speed alone cannot.  We can try updating the info text to make their definitions clearer.  I agree they are "second order" characteristics, so we can think about how to make them less prominent than wait and speed.  We could do bar chart for 90% and median times (for trip summary), then 3 cards for trip score/wait/speed (for trip summary and route summary), then 2 less prominent cards for long wait and travel variance (again, for trip summary and route summary).
